pkg/sql/logictest/testdata/logic_test/on_update

```diff
@@ -405,3 +405,48 @@ ALTER TYPE test_enum DROP VALUE 'x'
 
 statement error pq: could not remove enum value "y" as it is being used in an ON UPDATE expression of "test_array_enum"
 ALTER TYPE test_enum DROP VALUE 'y'
+
+# Interaction with backfills over row
```

Comment:
let's make the comment "Tests adding new columns will not trigger the ON UPDATE clause."

Reply:
Thanks! Done.